src/main/java/com/fitwsarah/fitwsarah/authenticationsubdomain/presentationlayer/SecurityConfig.java

```diff
@@ -80,7 +80,7 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
                 .authorizeHttpRequests(authz -> authz
                         .requestMatchers("api/v1/fitnessPackages/**").permitAll()
                         .requestMatchers("api/v1/appointments").permitAll()
-                        .requestMatchers("api/v1/feedbacks").permitAll()
+                        .requestMatchers("api/v1/feedbacks/**").permitAll()
```

Comment:
Are u sure we want everybody to be able to access the feedback maybe make then for all the role and not everyone